src/utils/openmc/summary.py

```diff
@@ -533,6 +532,14 @@ def _read_tallies(self):
 
 
     def make_opencg_geometry(self):
+
+        try:
+            from openmc.opencg_compatible import get_opencg_geometry
```

Comment:
Yeah I knew that this was breaking PEP8, but I think this is a better way to keep OpenCG an optional dependency than some other rather contrived way of tracking whether a user has OpenCG installed.